gui/app/templates/interactive/effect-options/custom-script.html

```diff
@@ -0,0 +1,20 @@
+<div class="effect-setting-container">
+    <div class="effect-specific-title"><h4>Which script would you like to use?</h4></div>
+    <div class="effect-info alert alert-info">
+        Place scripts in the <a id="scriptFolderBtn" href="#" ng-click="openScriptsFolder()">scripts folder</a> of the root Firebot directory, then refresh the dropdown.
```

Comment:
We will want to update the copy for this since it still refers to the "root" directory. Maybe something like:

"Place scripts in the scripts folder in Firebot's /user-settings directory, then refresh the dropdown."

For the <a> tag, we will probably want to add an underline to this link since this the blue info background makes it hard to tell its clickable: `style="text-decoration:underline"`